### PR TITLE
Preheat and healthcheck

### PR DESCRIPTION
**Pull request by kofj**
Add preheat APIs, handlers.

1. Manual preheat.
2. Instance health check.

Signed-off-by: fanjiankong <fanjiankong@tencent.com>

### CODE REVIEW

File: api/v2.0/swagger.yaml [hunk 1]
```diff
@@ -667,6 +667,33 @@ paths:
...
+          description: The JSON object of instance.
+          required: true
+          schema:
+            $ref: '#/definitions/Instance'
+      tags:
+        - preheat
+      responses:
+        '200':
+          description: Instance is healthy.
+        '400':
```
Comment (steven-zou) [inline api/v2.0/swagger.yaml:689]: Why not use the ref `$ref: '#/responses/401'`?

File: src/server/v2.0/handler/preheat.go [hunk 2]
```diff
@@ -248,6 +251,59 @@ func (api *preheatAPI) ListPolicies(ctx context.Context, params operation.ListPo
...
+	if err != nil {
+		return api.SendError(ctx, err)
+	}
+
+	policy, err := api.preheatCtl.GetPolicyByName(ctx, project.ProjectID, params.PreheatPolicyName)
+	if err != nil {
+		return api.SendError(ctx, err)
+	}
+
+	_, err = api.enforcer.EnforcePolicy(ctx, policy.ID)
```
Comment (steven-zou) [inline src/server/v2.0/handler/preheat.go:266]: I suggest accepting the returned execution id and return it to the front end with a proper format:

  - Option1: JSON data
  - Option2: add a location header in the response

File: src/server/v2.0/handler/preheat.go [hunk 3]
```diff
@@ -22,6 +23,7 @@ func newPreheatAPI() *preheatAPI {
 	return &preheatAPI{
 		preheatCtl: preheatCtl.Ctl,
 		projectCtl: projectCtl.Ctl,
+		enforcer:   preheatCtl.NewEnforcer(),
```
Comment (chlins) [inline src/server/v2.0/handler/preheat.go:26]: How about add a global enforcer like other controllers？

File: api/v2.0/swagger.yaml (hunk 1 above)
Comment (kofj) [reply api/v2.0/swagger.yaml:689]: I'm missed it, fix later.

File: src/server/v2.0/handler/preheat.go (hunk 3 above)
Comment (kofj) [reply src/server/v2.0/handler/preheat.go:26]: `mmpei` add a global enforcer at  #12428, I'll rebase and use it.

File: src/server/v2.0/handler/preheat.go (hunk 2 above)
Comment (kofj) [reply src/server/v2.0/handler/preheat.go:266]: Great, I'll return JOSN data include ID.

**Pull request merged**